src/manager.cc

```diff
@@ -272,6 +277,23 @@ bool fastcat::Manager::Process()
     jsd_write(it->second);
   }
 
+  // for each JSD context, pop their queues and push them onto the single 
+  // fastcat queue
+  SdoResponse entry;
+  for (auto it = jsd_map_.begin(); it != jsd_map_.end(); ++it) {
+    while(jsd_sdo_pop_response_queue(it->second, &entry.response)){
+      
+      if(entry.response.slave_id < *(it->second->ecx_context.slavecount) ){
+        entry.device_name = it->second->slave_configs[entry.response.slave_id].name;
+      }else{
+          entry.device_name = "invalid name";
```

Comment:
@alex-brinkman, I think `entry.response.success` should be set to `false` here. When processing the responses, it is better to rely on the boolean flag to determine whether the response is valid.